Documentation/Assignment 3/Assignment 3 Answers/tex/SEE_3.tex

```diff
@@ -1,4 +1,9 @@
 \section{Question 3}
 
-''3. Enumerate other 3 factors that could have been studied in the paper and why you think they would
-belong to good/bad practice (4 pts).''
\ No newline at end of file
+``Enumerate 3 other factors that could have been studied in the paper and why you think they would belong to good/bad practice (4 pts).'' \\
+
+\begin{enumerate}
+	\item \textbf{Test Driven Development}: This would probably a good practice. Although Test Driven Development might take more time and could therefore maybe end up in the `Cost over Time'-quarter, it would probably end up as a Good Practice. This is because the developed software will be easier maintainable and will be easier to verify for bugs. Because of that the quality of the software will improve, which will mean it will costs relatively less than when bugs have to removed at the end of development for example. Test Driven Development also means investing a little more time in testing while developing software. This could result in a longer duration, but we are convinced that this little investment of time will result in lower overall duration of the project, since bugs will be found easier and therefore also prevented easier. 
```

Comment:
This would probably a good practice, it is a good practice and the sentence doesn't make sense.